ratis-grpc/src/main/java/org/apache/ratis/grpc/GrpcTlsConfig.java

```diff
@@ -81,6 +88,18 @@ public File getTrustStoreFile() {
         .orElse(null);
   }
 
+  public TrustManager getSslTrustManager() {
+    return Optional.ofNullable(getTrustManager())
+        .map(TrustManagerConf::getTrustManager)
+        .orElse(null);
+  }
+
+  public KeyManager getSslKeyManager() {
+    return Optional.ofNullable(getKeyManager())
+        .map(KeyManagerConf::getKeyManager)
+        .orElse(null);
+  }
```

Comment:
Let's don't add new API to `GrpcTlsConfig`, which will be deprecated soon.  We should use the API from `TlsConf`.